.circleci/config.yml

```diff
@@ -285,6 +296,13 @@ workflows:
           requires:
             - npm-promotions
             - promote-channels
+
+      # verify the npm tags and installers are on the right version
+      - promote-verify:
```

Comment:
@mshanemc would it be better if close ctc were run last, just in case verify fails?

Reply:
I thought about that for a bit.  At that point, we already did the promotions, so I think CTC should be marked as closed (read as "the release completed"), even if the promotions don't verify for some reason.

The other thought here is it's hard to test these and I'm imagining a scenario where the promotion goes fine (usually does) but some script error causes the verification to fail, at least the first few times I try it.